src/main/kotlin/dev/minn/jda/ktx/interactions/utils.kt

```diff
@@ -23,7 +23,7 @@ import net.dv8tion.jda.api.entities.User
 import net.dv8tion.jda.api.interactions.commands.OptionType
 
 inline fun <reified T> optionType() = when(T::class.java) {
-    Int::class.java -> OptionType.INTEGER
+    Integer::class.java -> OptionType.INTEGER
```

Comment:
We should probably just make this multiple matches to catch all possible int types:

```suggestion
    Integer::class.java, java.lang.Long::class.java -> OptionType.INTEGER
```